probe-rs/src/bin/probe-rs/cmd/run.rs

```diff
@@ -172,9 +172,18 @@ fn run_loop(
         // the core printed before halting, such as a panic message.
         match core.status()? {
             probe_rs::CoreStatus::Halted(HaltReason::Breakpoint(BreakpointCause::Semihosting(
-                SemihostingCommand::Unknown { operation },
+                SemihostingCommand::Unknown(UnknownCommandDetails {
+                    operation,
+                    parameter,
+                }),
             ))) => {
-                tracing::error!("Target wanted to run semihosting operation {:#x}, but probe-rs does not support this operation yet. Continuing...", operation);
+                tracing::warn!("Target wanted to run semihosting operation {:#x} with parameter {:#x}, but probe-rs does not support this operation yet. Continuing...", operation, parameter);
+                core.run()?;
+            }
+            probe_rs::CoreStatus::Halted(HaltReason::Breakpoint(BreakpointCause::Semihosting(
+                SemihostingCommand::GetCommandLine(_),
```

Comment:
If we don't support this yet, why does this variant exist as a special case instead of just going with unknown?

Reply:
We'll use `SYS_GET_CMDLINE` heavily for embedded-test... I did a separate PR, because this PR contains an API breaking change.

Reply:
Okii, thanks!